.github/workflows/sync-python-releases.yml

```
@@ -1,7 +1,7 @@
-# Sync Python releases from `python-build-standalone`.
+# Sync Python releases and create a pull request.
 #
 # Based on: https://github.com/astral-sh/rye/blob/57b7c089e494138aae29a130afb2e17f447970bf/.github/workflows/sync-python-releases.yml
-name: python-build-standalone
+name: "Sync Python Releases"
```

Fwiw we don't capitalize all the words in most of our other jobs with names like this.

Wdyt of "Sync Python downloads"?

```suggestion
name: "Sync Python downloads"
```